torch/_torch_docs.py

```diff
@@ -4338,6 +4338,48 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(self, other, *, out=None) -> Tensor
+
+Computes ``self * log(other)`` except it treats ``0 * log(0)`` as ``0``.
```

Comment:
I think we looked at this briefly before, but I don't think we can say "it treats..." because if `self` is zero then all values of `log`, including negative values, will return 0. And PyTorch treats the logarithm of negative numbers as nan, too:

```
t = torch.tensor((-2., -1, 0, 1, 2))
torch.log(t)
: tensor([   nan,    nan,   -inf, 0.0000, 0.6931])
```

The problem with the "it treats" clause here is that this implies it's the ONLY exception from the normal computation. 

What about saying, "Computes \`\`self * log(other)\`\` but with the following special cases:"

Then putting the mathematical cases below that line, then the sentence about this being similar to SciPy's xlogy after that?